.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,50 @@
+name: Feature request
+description: Suggest an idea for something we could add or improve
+title: "[Feature Request]: "
+labels: [enhancement, triage]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for taking the time to suggest a new feature!
+  - type: markdown
+    attributes:
+      value: Please describe the feature request in detail. Help us understand why it would be a good addition to cssnano.
+  - type: textarea
+    id: description
+    attributes:
+      label: What should be improved?
+      description: Is your feature request related to a problem? Please describe it.
+      placeholder: Ex. `foo-bar` is not minimised
+    validations:
+      required: true
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you would like
+      description: A clear and concise description of what you want to happen.
+      placeholder: Ex. reduce `foo-bar` to the `foo` shorthand. This works in all browsers.
+    validations:
+      required: true
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Possible alternative solutions
+      description: Describe any alternative solutions/approaches you have considered.
+      placeholder: Ex. it could also be replaced by `baz`, but that is potentially unsafe in chrome.
+    validations:
+      required: false
+  - type: textarea
+    id: extra
+    attributes:
+      label: Additional context
+      description: Add any other information/context that might be relevant here.
+      placeholder: Ex. It could be a good idea to double check that `foo` works in Edge and IE.
+    validations:
+      required: true
+  - type: checkboxes
+    id: can-work
+    attributes:
+      label: Are you willing to work on this?
+      description: Are you willing to help us add this feature? If you are not sure how, then please feel free to ask for guidance.
```

Comment:
I am not sure it's clear how someone can ask for guidance? I really don't know either. Should they start a discussion?